problema1_test

```diff
@@ -0,0 +1,36 @@
+require 'test/unit'
+require_relative "problema1"
+
+class MyTest < Test::Unit::TestCase
+
+  # Called before every test method runs. Can be used to set up fixture information.
+  def setup
+    # Do nothing
+  end
+
+  # Called after every test method runs. Can be used to tear down fixture information.
+  def teardown
+    # Do nothing
+  end
+
+  def test_simplu
```

Comment:
The problem text in in English it will be good to preserve the language in the solutions and tests.